src/encoded/docs/schema-changes.md

```diff
@@ -154,69 +154,141 @@ This new object is an array of example objects that can successfully POST agains
+
+Up until schema version 6 for the genetic modifications object, one of the possible enums for "purpose" property was "validation". As a part of schema version upate to 7, "validation" was removed and instead the term "characterization" was added.
+
 
-1. In the **schemas** directory, increment the schema version (1->2), if it meets the above rule: 
+        "purpose":{
+            "title": "Purpose",
+            "description": "The purpose of the genetic modification.",
+            "type": "string",
+            "enum": [
+                "activation",
+                "analysis",
+                "overexpression",
+                "repression",
+                "tagging",
+                "validation",
+                "screening",
+                "expression"
+            ]
+        },
+        # Replacing the enum "validation" by the enum "characterization" in the list of enums of the "purpose" property:
+        "purpose":{
+            "title": "Purpose",
+            "description": "The purpose of the genetic modification.",
+            "type": "string",
+            "enum": [
+                "activation",
+                "analysis",
+                "overexpression",
+                "repression",
+                "tagging",
+                "characterization",
+                "screening",
+                "expression"
+            ]
+        },
+
+2. In the **schemas** directory, edit the existing properties in the corresponding JSON file named after the object and increment the schema version.
+
+**Specific example from the genetic modifications object upgrade:**
+
+For example if the original schema version for the genetic modification object being modified was "6", change it to "7" (6->7): 
         
         "schema_version": {
-            "default": "2"
+            "default": "7"
         }
 
-2. In the **upgrade** directory add an ```upgrade_step``` to an existing/new python file named after the object. An example upgrades the object from schema version (2->3):
+3. In the **upgrade** directory add an ```upgrade_step``` to a python file named after the object (create new if there is no such a file in the upgrade directory". For some objects the upgrade happens in the abstract class - like dataset for the experiment.
 
-        @upgrade_step('{metadata_object}', '2, '3')
-        def {metadata_object}_2_3(value, system):
-            # {Redmine Issue}
+**Specific example from the genetic modifications object upgrade:**
 
-            # lowercase all values in property 1
-            if 'property_1' in value:
-                value['property_1'] = value['property_1'].lower()
+An example to the upgrade step is shown below. Continuing with our example on genetic modifications, all the existing objects with that had "purpose" specified to be "validation" must now be changed to "characterization". And since the schema is changing from version 6 to 7 the def must specify this (6->7):
```

Comment:
Please remove "And since the schema is changing from version 6 to 7 the def must specify this (6->7)"